src/plugins/dashboard/public/application/top_nav/options.tsx

```diff
@@ -101,12 +111,24 @@ export class OptionsMenu extends Component<Props, State> {
           />
         </EuiFormRow>
 
+        <EuiFormRow>
+          <EuiSwitch
+            label={i18n.translate('dashboard.topNav.options.syncCursorBetweenPanelsSwitchLabel', {
+              defaultMessage: 'Sync cursor across panels',
+            })}
+            checked={this.state.syncCursor}
+            onChange={this.handleSyncCursorChange}
+            data-test-subj="dashboardSyncCursorCheckbox"
+          />
+        </EuiFormRow>
+
         <EuiFormRow>
           <EuiSwitch
             label={i18n.translate('dashboard.topNav.options.syncTooltipsBetweenPanelsSwitchLabel', {
               defaultMessage: 'Sync tooltips across panels',
             })}
             checked={this.state.syncTooltips}
+            disabled={!Boolean(this.state.syncCursor)}
```

Comment:
Nice touch! If your cursor isn't synced, doesn't make sense that the tooltip **is** synced :+1: